### PR TITLE
qtpass: Add version 1.3.2

### PR DESCRIPTION
**Pull request by ChungZH**

### CODE REVIEW

File: bucket/qtpass.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "bin": "qtpass.exe",
+    "shortcuts": [
+        [
+            "qtpass.exe",
+            "QtPass"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/IJHack/QtPass/",
+        "regex": "tag/v([\\w.-]+)"
```
Comment (Ash258) [inline bucket/qtpass.json:18]: There is no tag with `-` or any other character than `\d`

File: bucket/qtpass.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://raw.githubusercontent.com/lukesampson/scoop/master/schema.json",
```
Comment (Ash258) [inline bucket/qtpass.json:2]: Did you see any other manifest with this??

File: bucket/qtpass.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "1.3.2",
+    "license": "GPL-3.0-only",
+    "homepage": "https://qtpass.org/",
```
Comment (Ash258) [inline bucket/qtpass.json:5]: Missing description

File: bucket/qtpass.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "bin": "qtpass.exe",
+    "shortcuts": [
+        [
+            "qtpass.exe",
+            "QtPass"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/IJHack/QtPass/",
+        "regex": "tag/v([\\w.\\d]+)"
```
Comment (Ash258) [inline bucket/qtpass.json:18]: ...

- `\d` is contained in `\w`
- Why would you use custom regex, when this release is covered by the native regex??

```suggestion
        "github": "https://github.com/IJHack/QtPass",
```

Comment (ChungZH) [reply bucket/qtpass.json:18]: ahhhh I faint 😂

File: bucket/qtpass.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.3.2",
+    "license": "GPL-3.0-only",
+    "homepage": "https://qtpass.org/",
+    "description": "GUI for pass, the standard UNIX password manager.",
```
Comment (Ash258) [inline bucket/qtpass.json:5]:
```suggestion
    "description": "GUI for standard UNIX password manager",
    "homepage": "https://qtpass.org",
    "license": "GPL-3.0-only",
```

File: bucket/qtpass.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "url": "https://github.com/IJHack/QtPass/releases/download/v1.3.2/qtpass-1.3.2.exe",
+    "hash": "0fb9e68ce7e52e614ebb29dd9fc0b06b24f4262e9509090853e2c2991c171e72",
+    "innosetup": true,
```
Comment (Ash258) [inline bucket/qtpass.json:8]: https://github.com/IJHack/QtPass/blob/d7d7443ad8e607ac0f483984bf8814f6f4aa83d0/src/qtpasssettings.cpp#L18-L30

```suggestion
    "innosetup": true,
    "pre_install": [
        "ensure \"$persist_dir\" | Out-Null",
        "if (Test-Path \"$persist_dir\\qtpass.ini\") {",
        "    Copy-Item \"$persist_dir\\qtpass.ini\" \"$dir\" -ErrorAction SilentlyContinue",
        "} else {",
        "    New-Item \"$dir\\qtpass.ini\" | Out-Null",
        "}"
    ],
    "uninstaller": {
        "script": "Copy-Item \"$dir\\qtpass.ini\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
    },
```

**Pull request closed without merging**